### PR TITLE
add ontology network

### PR DESCRIPTION
**Pull request by lucas7788**
Fixes # .

Changes proposed in this pull request:
- 
- 
-

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -875,6 +890,23 @@
...
+    "chainId": 5851,
+    "network": "testnet",
+    "multicall": "0x381445710b5e73d34aF196c53A3D5cDa58EDBf7A",
+    "rpc": [
+      "https://polaris1.ont.io:10339",
+      "https://polaris2.ont.io:10339",
+      "https://polaris3.ont.io:10339",
+      "https://polaris4.ont.io:10339",
+      "https://polaris4.ont.io:10339",
+      "https://polaris5.ont.io:10339"
```
Comment (ChaituVR) [inline src/networks.json:905]: All these nodes don't work? only the first one works perfectly.

File: src/networks.json [hunk 2]
```diff
@@ -875,6 +890,23 @@
     "start": 53370,
     "imageIPFS": "QmPXPCBho3kGLt5rhG9JGkKmzdtLvqZmJqGzzijVCuggWY"
   },
+  "5851": {
+    "key": "5851",
+    "name": "Ontology Testnet",
+    "chainId": 5851,
+    "network": "testnet",
+    "multicall": "0x381445710b5e73d34aF196c53A3D5cDa58EDBf7A",
```
Comment (ChaituVR) [inline src/networks.json:898]: I can't check the multicall code here https://explorer.ont.io/testnet/contract/other/0x381445710b5e73d34aF196c53A3D5cDa58EDBf7A/10/1 

Is there a way to check the source code of the contract?

File: src/networks.json [hunk 3]
```diff
@@ -266,6 +266,21 @@
...
+    "key": "58",
+    "name": "Ontology Mainnet",
+    "chainId": 58,
+    "network": "mainnet",
+    "multicall": "0xce6292279bf688173B269Df080E14407470A9E60",
+    "rpc": [
+      "https://dappnode1.ont.io:10339",
+      "https://dappnode2.ont.io:10339",
+      "https://dappnode3.ont.io:10339",
+      "https://dappnode4.ont.io:10339"
```
[the remark this answers is not in the document]
Comment (lucas7788) [reply src/networks.json:279]: Could you please tell me which method you called?

File: src/networks.json (hunk 1 above)
Comment (lucas7788) [reply src/networks.json:905]: Could you please tell me which method you called?

File: src/networks.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (ChaituVR) [reply src/networks.json:279]: Hey @lucas7788 I was calling getBlockNumber from ethers js, from chrome browser, (maybe it blocks requests from different domains? ) 
```js      
      provider = new StaticJsonRpcProvider(URL)
      latestBlockNumber = await provider.getBlockNumber();
```

[the remark this answers is not in the document]
Comment (lucas7788) [reply src/networks.json:279]: Thanks, I'll look into the problem

[the remark this answers is not in the document]
Comment (lucas7788) [reply src/networks.json:279]: we have fixed this problem, please take a look again.

[the remark this answers is not in the document]
Comment (lucas7788) [reply src/networks.json:279]: > we have fixed this problem, please take a look again.

we have fixed this problem, please take a look again.

File: src/networks.json (hunk 2 above)
Comment (ChaituVR) [reply src/networks.json:898]: @lucas7788 Can you plz check this too plz

File: src/networks.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (ChaituVR) [reply src/networks.json:279]: Nodes work well now 👍

File: src/networks.json (hunk 2 above)
Comment (blockchainer2) [reply src/networks.json:898]: we have fixed this problem, please tale a look again.

**Review by bonustrack — APPROVED** [review]

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**